src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -49,4 +49,40 @@ def progress(staging_project)
       "#{testing_progress(staging_project)} %"
     end
   end
+
+  def requests(staging_project)
+    number_of_requests = staging_project.classified_requests.size
+
+    return 'None' if number_of_requests == 0
+
+    requests_visible_by_default = 10
+    requests_links = staging_project.classified_requests.map do |request|
+      css = 'ready'
+      css = 'review' if request[:missing_reviews].present?
+      css = 'obsolete' if request[:state].in?(BsRequest::OBSOLETE_STATES)
+
+      link_to(request_show_path(request[:number]), class: 'request') do
+        content_tag(:span, request[:package], class: "badge state-#{css}")
+      end
+    end
+
+    if number_of_requests <= requests_visible_by_default
+      return safe_join(requests_links)
+    end
+
+    output = safe_join(requests_links[0, requests_visible_by_default])
+
+    # Add requests links which aren't visible by default in a collapsible div controlled by a clickable icon
+    # TODO: Add the 'collapse' molecule to obs-patterns
```

Comment:
[nitpick] I would create an issue in that project, instead of here in the code. There is nothing to do in the code :unamused: